src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/FlyoutSamplePage.xaml

```diff
@@ -0,0 +1,166 @@
+﻿<Page x:Class="Uno.Material.Samples.Content.Controls.FlyoutSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:not_win="http://uno.ui/not_win"
+	  xmlns:ios="http://uno.ui/ios"
+	  xmlns:android="http://uno.ui/android"
+	  xmlns:wasm="http://uno.ui/wasm"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:extensions="using:Uno.Material.Extensions"
+	  xmlns:controls="using:Uno.Material.Controls"
+	  xmlns:toolkit="using:Uno.UI.Toolkit"
+	  mc:Ignorable="not_win android ios wasm">
+
+	<ScrollViewer VerticalScrollBarVisibility="Auto">
+		<StackPanel BorderBrush="{StaticResource MaterialOnSurfaceBrush}"
+					BorderThickness="1"
+					android:Margin="12"
+					ios:Margin="12"
+					Padding="12">
+
+			<!-- Flyout Text -->
+			<TextBlock Text="Flyout"/>
+
+			<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Flyout">
+
+				<Button Content="Flyout">
+					<Button.Flyout>
+						<Flyout FlyoutPresenterStyle="{StaticResource MaterialFlyoutPresenterStyle}">
+							<TextBlock Text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+									   Style="{StaticResource MaterialBody2}" />
+						</Flyout>
+					</Button.Flyout>
+				</Button>
+			</smtx:XamlDisplay>
+
+			<!-- Flyout Text -->
+			<TextBlock Text="MenuFlyout &amp; Icons"
+					   Margin="0,16,0,0"/>
+
+			<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Flyout_Icons">
+
+				<Button Content="Open">
+					<Button.Flyout>
+						<MenuFlyout MenuFlyoutPresenterStyle="{StaticResource MaterialMenuFlyoutPresenterStyle}">
+
+							<MenuFlyoutItem Text="Like"
+											Style="{StaticResource MaterialMenuFlyoutItemStyle}">
+								<MenuFlyoutItem.Icon>
+									<SymbolIcon Symbol="Like" />
+								</MenuFlyoutItem.Icon>
+							</MenuFlyoutItem>
+
+							<MenuFlyoutItem Text="Dislike"
+											Style="{StaticResource MaterialMenuFlyoutItemStyle}">
+								<MenuFlyoutItem.Icon>
+									<SymbolIcon Symbol="Dislike" />
+								</MenuFlyoutItem.Icon>
+							</MenuFlyoutItem>
+
+							<MenuFlyoutSeparator Style="{StaticResource MaterialMenuFlyoutSeparatorStyle}" />
+
+							<MenuFlyoutItem Text="Calculator"
+											Icon="Calculator"
+											Style="{StaticResource MaterialMenuFlyoutItemStyle}" />
+
+							<MenuFlyoutItem Text="Camera"
+											Icon="Camera"
+											KeyboardAcceleratorTextOverride="Ctrl+C"
+											Style="{StaticResource MaterialMenuFlyoutItemStyle}" />
+
+							<MenuFlyoutItem Text="Calendar"
+											Icon="Calendar"
+											IsEnabled="False"
+											KeyboardAcceleratorTextOverride="Ctrl+C"
```

Comment:
Don't re-use the same keyboard hooks.
Probably shouldn't even be included in this sample, since only 2 platforms can support this properly.

Reply:
I could make them visible only on UWP and WASM

Reply:
Not sure they work on WASM, do they? I think Xiao was referring to macos and UWP. And MAcOS won't even be "ctrl", it will be cmd =D I don't think it's implemented.

@ajpinedam @jeromelaban @MartinZikmund any of you know the status of keyboard shortcuts in menu flyouts?

Reply:
I suggest we just remove them from the samples entirely

Reply:
Accelerators are partially working, yes on platforms that have keyboard support. Not sure if Ctrl+C completely works, but it will soon.